src/models/contact/group/index.test.js

```diff
@@ -99,6 +100,18 @@ const groupIndexWithGroup1And2Dataset = chain(
       group2Url
     )
 );
+const group1Thing = getThing(groupIndexWithGroup1And2Dataset, group1Url);
+const groupIndexAfterRemovingGroup1 = chain(
+  groupIndexWithGroup1And2Dataset,
+  (d) =>
+    removeGroupFromMockedIndexDataset(
```

Comment:
I'm probably lacking a bit too much context here - could you maybe clarify how the changes in solid-ui-react require the changes in this file, as well as the change in the addressBook's test in which `3` suddenly had to be replaced by `4`, which I suppose is related?

Reply:
I found that a lot of the order in which the datasets/things extracted from datasets were being returned changed. I assumed this could be related to `solid-client` but I don't know for sure.

Reply:
Hmm, weird. I'm not sure what could cause this, but since the order doesn't matter for the actual code, probably not worth spending any time on diving into that.